aws/resource_aws_appsync_graphql_api_test.go

```diff
@@ -44,6 +44,40 @@ func TestAccAWSAppsyncGraphqlApi_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSAppsyncGraphqlApi_schema(t *testing.T) {
+	rName := acctest.RandomWithPrefix("tf-acc-test")
+	resourceName := "aws_appsync_graphql_api.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAwsAppsyncGraphqlApiDestroy,
+		Steps: []resource.TestStep{
```

Comment:
Since the new attribute is not marked `ForceNew: true` an additional `TestStep` should be included to ensure updates occur correctly.